middleware/file/setup_test.go

```diff
@@ -0,0 +1,75 @@
+package file
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/mholt/caddy"
+)
+
+func TestFileParse(t *testing.T) {
+	zonePath, err := ioutil.TempDir("", "TestFilePath")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(zonePath)
+
+	zoneFile1, err := ioutil.TempFile(zonePath, "zone1-")
+	if err != nil {
+		t.Fatal(err)
+	}
+	zoneFileName1 := zoneFile1.Name()
+	zoneFile1.Write([]byte(dbMiekNL))
```

Comment:
prolly slowly move off using `miek.nl` for these kinds of tests and start using example.{org,net,com}